### DISCUSSION TITLE
Anyone still here?

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#130] [General (Friday Night Chattin')]
I’m back, but only just to actually finish what I started, being this very update

After this, I’m gonna pull a PR to Theo’s engine with some of my code, and then I won’t be doing anymore stuff with Psych Engine

that’s it, cause the Engine’s community has obscured the shit out of source, nobody wants to learn, and would rather steal scripts from experienced programmers than learn for themselves using a language that is VERY outdated and is seen frequently in the modding scene of FNF, being Lua.

I’m tired of this, from here on out, after doing everything as said above, I’ll be trying out other engines for modding, cause come on, you gotta eventually get out of your comfort zone and try something new.

If you’re some toddler who’s a fanatic of Psych Engine and is only reading the bad shit and not hearing me out…

### Get that ugly ass mindset out, nobody likes an android porter or lua kid, it’s about time you start growing up.

As for the rest of you who HAVE read everything and is in full understanding of this situation…

### Thank you.

It was fun while it lasted, wished the Psych Engine community didn’t get this toxic and overran with stupid ass Dave & Bambi shit in the first place.

**Comment by jasonroni** [comment]
Well everyone isn’t here RIP

**Comment by Theoyeah** [comment]
I respect, one day if fnf still exists, i'll create my real own engine...
